services/workflows-service/src/business/business.service.ts

```diff
@@ -66,6 +68,105 @@ export class BusinessService {
     return await this.repository.updateById(id, args, transaction);
   }
 
+  async getMerchantMonitoringMetrics({
+    projectIds,
+    features,
+    from,
+    to,
+  }: {
+    projectIds: string[];
+    features: TCustomerWithFeatures['features'];
+    from: string | undefined;
+    to: string | undefined;
+  }): Promise<{
+    totalActiveMerchants: number;
+    addedMerchantsCount: number;
+    unmonitoredMerchants: number;
+  }> {
+    // Metrics are currently mostly requested by month
+    if (!from) {
+      from = dayjs().startOf('month').toISOString();
+    }
+
+    if (!to) {
+      to = dayjs(from).add(1, 'month').toISOString();
+    }
+
+    const allProjectMerchants = await this.repository.findMany({}, projectIds);
+
+    const totalActiveMerchants = allProjectMerchants.filter(b => {
+      const disabledAt = z
+        .number()
+        .nullable()
+        .catch(() => null)
+        .parse(
+          (
+            b.metadata as {
+              featureConfig: Record<
+                (typeof FEATURE_LIST)[keyof typeof FEATURE_LIST],
+                TCustomerFeaturesConfig & { disabledAt: number | null | undefined }
+              >;
+            }
+          )?.featureConfig?.[FEATURE_LIST.ONGOING_MERCHANT_REPORT]?.disabledAt,
+        );
+
+      return (
+        disabledAt === null ||
+        (b.metadata === null && features?.ONGOING_MERCHANT_REPORT?.options?.runByDefault)
+      );
```

Comment:
I'm pretty sure this would work as well, I think it's a bit more readable, WDYT?
```suggestion
      const isEnabled = b?.metadata?.featureConfig?.[FEATURE_LIST.ONGOING_MERCHANT_REPORT]?.enabled,
        );

      return (
        isEnabled === false ||
        (isEnabled === undefined && features?.ONGOING_MERCHANT_REPORT?.options?.runByDefault)
      );
```